### ISSUE TITLE
Error loading css from drag-and-drop.js

### ISSUE THREAD

**Issue by spapaseit** [michaelmalonenz/aurelia-dragula#16]
I've just installed `aurelia-dragula` (version 1.2.0).

After adding `aurelia.use.plugin("aurelia-dragula")` to `main.js` I started my app but I get the following error: 

    /jspm_packages/npm/aurelia-dragula@1.2.0/dragula.css!http://localhost:49849/dist/css.js as 
    "./dragula.css!css" from http://localhost:49849/jspm_packages/npm/aurelia-dragula@1.2.0/dragula-and-drop.js

and the application stops loading.

It looks like it tries to load `dragula.css` from the wrong location. Any suggestions?

**Comment by michaelmalonenz** [comment]
TL;DR: `jspm install css`

The long version:
Sorry - I changed the dependencies in v1.2.0 as it now requires the css plugin for systemjs to be installed (`jspm install css`).  I tried to list this as a proper dependency, but jspm wasn't happy about it; despite me using it to list the dependency in the first place...

Anyway, I will try to fix this soon - I think I understand the problem that caused me to use this style of import a bit better and hope to get it fixed or worked around soon (it is to do with the changes in Aurelia for the Shadow DOM).  So even though installing the css plugin will work as a temporary solution, please leave this issue open and I will close it once I no longer depend on systemjs' plugins.

**Comment by spapaseit** [comment]
Thanks for the explanation, I appreciate it.

**Comment by spapaseit** [comment]
By the way, I know this is not the place to ask, but do you happen to have an example on how to reorder table rows? `tr`s get reverted to the original position every time I drop them

**Comment by michaelmalonenz** [comment]
v1.2.1 Fixes this issue - evidently when I was upgrading to the pre-release-candidate builds, there was a bug that was not allowing my require in the inline view to be the only element of consequence.  That appears to be fixed with the release candidate.  I've fixed my code and removed the jspm css plugin dependency
